ci-operator/step-registry/user-setting/htpasswd/user-setting-htpasswd-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ ref:
+  as: user-setting-htpasswd
```

Comment:
For the folder name `user-setting`, how about use `idp` or `identity-providers` ?

Reply:
@xingxingxia What do you think?

Reply:
Good suggestion.